mesonbuild/modules/python.py

```diff
@@ -280,7 +280,21 @@ def set_env(name, value):
 import sysconfig
 import json
 import sys
-import distutils.command.install
+
+def debian_distutils_missing():
+    # Debian partially splits the distutils module from the python3, they
+    # keep distutils and distutils.version in python3 and put the rest of
+    # the submodules in python3-distutils
+    try:
+        import distutils
+        import distutils.version
```

Comment:
The second import includes the first import, we only need the second one.

Reply:
The check could be even simpler, I think.
```python
try:
    import distutils.version
    import distutils.core
except ModuleNotFoundError as e:
    if e.name == 'distutils.core':
        return True # distutils was found, and distutils.version successfully imported, but core did not
return False
```

Also ModuleNotFoundError > ImportError here, since we specifically want to detect "a module could not be located" rather than "the import statement has troubles trying to load a module"